### PR TITLE
optuna-examples #11 follow up

### PR DESCRIPTION
**Pull request by hvy**
## Motivation

Follow up to https://github.com/optuna/optuna-examples/pull/11. 

## Description of the changes

Add URL to new example.

### CODE REVIEW

**Comment by hvy** [comment]
@Crissman , @c-bata could you review this change?

**Review by c-bata — APPROVED** [review]
LGTM 👍

**Review by Crissman — APPROVED** [review]
LGTM

File: examples/README.md [hunk 1]
```diff
@@ -65,6 +65,10 @@ In addition, integration modules are available for the following libraries, prov
 * [Pruning with XGBoost integration module](https://github.com/optuna/optuna-examples/blob/main/xgboost/xgboost_integration.py)
 * [Pruning with XGBoost integration module (cross validation, XGBoost.cv)](https://github.com/optuna/optuna-examples/blob/main/xgboost/xgboost_cv_integration.py)
 
+### Examples of Samplers
+
+* [Warm Starting CMA-ES](https://github.com/optuna/optuna-examples/blob/main/samplers/warm_starting_cma.py)
+
 ### Examples of User-Defined Sampler
 
 * [SimulatedAnnealingSampler](https://github.com/optuna/optuna-examples/blob/main/samplers/simulated_annealing_sampler.py)
```
Comment (Crissman) [inline examples/README.md:74]:
```suggestion
* [User-defined SimulatedAnnealingSampler](https://github.com/optuna/optuna-examples/blob/main/samplers/simulated_annealing_sampler.py)
```

File: examples/README.md [hunk 2]
```diff
@@ -65,6 +65,10 @@ In addition, integration modules are available for the following libraries, prov
 * [Pruning with XGBoost integration module](https://github.com/optuna/optuna-examples/blob/main/xgboost/xgboost_integration.py)
 * [Pruning with XGBoost integration module (cross validation, XGBoost.cv)](https://github.com/optuna/optuna-examples/blob/main/xgboost/xgboost_cv_integration.py)
 
+### Examples of Samplers
+
+* [Warm Starting CMA-ES](https://github.com/optuna/optuna-examples/blob/main/samplers/warm_starting_cma.py)
+
 ### Examples of User-Defined Sampler
```
Comment (Crissman) [inline examples/README.md:73]:
```suggestion
```

**Review by Crissman — CHANGES_REQUESTED** [review]
Actually, I think these two groups should be joined as a generic samplers.

**Comment by hvy** [comment]
I see, I don't have a strong opinion about it but sounds good as well being consistent with the file structure. I'd like to fix the inconsistency with `optuna-examples` in this PR however, could we have that discussion in a separate ticket?

**Review by Crissman — APPROVED** [review]
Will deal with merging samplers example in separate PR.

**Pull request merged**